Sources/SwiftDocC/Semantics/Symbol/Symbol.swift

```diff
@@ -414,6 +417,20 @@ extension UnifiedSymbolGraph.Symbol {
     var defaultSymbol: SymbolGraph.Symbol? {
         symbol(forSelector: defaultSelector)
     }
+    
+    /// Returns the primary symbol to use as documentation source.
+    var documentedSymbol: SymbolGraph.Symbol? {
```

Comment:
This API seems a bit odd to me. If it's the doc comment we're interested in, should the API not just return the doc comment rather than the whole symbol?

Reply:
Generally this was just what caused the least code churn. But I think this API  makes sense long-term as well. There are cases when we need all the metadata that comes with a symbol, but specifically the documented one.

For example, we might want to attach a diagnostic to the documented version of a symbol and needs it source location.